.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release
+
+env:
+  model_name: zeke/stable-diffusion
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.10"
+
+      - name: Install Python dependencies
+        run: |
+          pip install -r requirements.txt
+
+      - name: Download weights
+        run: |
+          script/download-weights
```

Comment:
This is documented in the README as `cog run script/download-weights`, but because this Cog model requires GPU, it's not possible to `cog run` anything from the Actions environment, where no GPU is present.

I've added the above steps to manually install Python and pip install stuff as a way of working around this.